cli/azd/pkg/project/framework_service_dotnet.go

```diff
@@ -49,6 +49,19 @@ func (dp *dotnetProject) InstallDependencies(ctx context.Context) error {
 	return nil
 }
 
+func (dp *dotnetProject) Initialize(ctx context.Context) error {
+	dotnetCli := tools.NewDotNetCli()
```

Comment:
I would make the `dotnetCli` a field on the struct so you don't need to create it in multiple methods now.